atmo/clusters/models.py

```diff
@@ -9,10 +9,79 @@
 from django.db import models
 from django.utils import timezone
 
-from ..models import CreatedByModel, EditedAtModel, EMRReleaseModel
+from ..models import CreatedByModel, EditedAtModel
 from .provisioners import ClusterProvisioner
 
 
+class EMRReleaseQuerySet(models.QuerySet):
+
+    def stable(self):
+        return self.filter(is_experimental=False, is_deprecated=False)
+
+    def experimental(self):
+        return self.filter(is_experimental=True)
+
+    def deprecated(self):
+        return self.filter(is_deprecated=True)
+
+
+class EMRRelease(EditedAtModel):
```

Comment:
What about adding a `is_active` attribute? is_active==False would be the final state of a EMRRelease object.